yogstation/code/modules/clothing/donator.dm

```diff
@@ -189,14 +189,26 @@
 	name = "Inmate belongings box"
 	desc = "A criminals' belongings, returned upon their release. Just stop killing people."
 
-/obj/item/storage/box/boxta/PopulateContents() 
+/obj/item/storage/box/boxta/PopulateContents()
 	. = ..()
 	new /obj/item/clothing/neck/falcon(src)
 	new /obj/item/clothing/head/fedora/gtrim_fedora(src)
 
+/obj/item/storage/box/somecustodian
+	name = "Golden Box"
+	desc = "Holds a set of one of the most powerful suits of armor known to mankind.(99 cents rip off)"
+
+/obj/item/storage/box/somecustodian/PopulateContents()
+	. = ..()
+	new /obj/item/clothing/head/custodianhelm(src)
+	new /obj/item/clothing/suit/custodianchest(src)
+
+/obj/item/bedsheets/pridebedsheet/cark
 /obj/item/bedsheet/pride
 	name = "Pride Bedsheet"
 	desc = "A very prideful queen size bedsheet. How you are carrying all that weight is beyond me though."
 	mob_overlay_icon = 'icons/mob/clothing/neck/sheets.dmi'
+	lefthand_file = 'icons/mob/inhands/misc/bedsheet_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/misc/bedsheet_righthand.dmi'
```

Comment:
ditto

Reply:
I dont know where it was supposed to go

Reply:
why are you touching the pride bedsheet at all

Reply:
it was a "conflict" and somehow some of my code was broken inside it? soo I guess removed the code that was mine

Reply:
remove the changed lines

Reply:
just totally remove them? alright